CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 ### Fixed
 
 - ([#858](https://github.com/demos-europe/demosplan-ui/pull/858)) DpCheckbox: fix vue warning related to type checking ([@sakutademos](https://github.com/sakutademos))
+- ([#850](https://github.com/demos-europe/demosplan-ui/pull/850)) Fix the validation of the relation when the 'data-dp-validate-if' is provided ([@sakutademos](https://github.com/sakutademos))
```

Comment:
I think this entry does not express very clearly what is going on. Maybe something like
`Input validation: Always use form as validation container`
or
`Input validation: Always use querySelector on form element`
is slightly better to understand?
(I like to add a context like "Input validation:" at the beginning of the message)

Reply:
right, done here https://github.com/demos-europe/demosplan-ui/pull/850/commits/ae879da1bed75f3ad12d621fa9801060d9eadc93